src/javascript/EditPanel/EditPanel.jsx

```diff
@@ -9,15 +9,23 @@ import {HeaderLowerSection, HeaderUpperSection} from './header';
 import {useContentEditorContext, useContentEditorConfigContext} from '~/ContentEditor.context';
 import {PublicationInfoContext} from '~/PublicationInfo/PublicationInfo.context';
 import {registry} from '@jahia/ui-extender';
+import {useNodeChecks} from '@jahia/data-helper';
 
 import MainLayout from '~/DesignSystem/ContentLayout/MainLayout';
 import ContentHeader from '~/DesignSystem/ContentLayout/ContentHeader';
 import {Separator} from '@jahia/moonstone';
+import {Constants} from '~/ContentEditor.constants';
 
 const EditPanelCmp = ({formik, title, notificationContext, client}) => {
     const {t} = useTranslation();
-    const {nodeData, siteInfo, lang, uilang, mode, nodeTypeName, refetchFormData} = useContentEditorContext();
+    const {nodeData, siteInfo, site, lang, uilang, mode, nodeTypeName, refetchFormData} = useContentEditorContext();
     const {envProps} = useContentEditorConfigContext();
+    const [activeTab, setActiveTab] = useState('edit');
+
+    const advancedTabPermission = useNodeChecks(
+        {path: `/sites/${site}`},
+        {requiredPermission: [Constants.permissions.canSeeAdvancedOptionsTab]}
+    );
```

Comment:
This has to be set at tab registration level, not hardcoded here.

Reply:
is there any doc how to use permissions with registry?

Reply:
or any similar working code?

Reply:
yes, you can find an example of an action with restriction:
https://github.com/Jahia/content-editor/blob/master/src/javascript/Create/CreateNewContentAction/createNewContent.action.jsx#L48

Reply:
The check has to be done in the advance option component

Reply:
I'll see